pkg/controller/policyrecommendation/policyrecommendation_controller.go

```diff
@@ -417,20 +394,38 @@ func (r *ReconcilePolicyRecommendation) Reconcile(ctx context.Context, request r
 
 		certificateManager.AddToStatusManager(r.status, helper.InstallNamespace())
 
-		policyRecommendationCfg.TrustedBundle = trustedBundle
-		policyRecommendationCfg.PolicyRecommendationCertSecret = policyRecommendationKeyPair
-
-		components = append(components,
-			rcertificatemanagement.CertificateManagement(&rcertificatemanagement.Config{
-				Namespace:       helper.InstallNamespace(),
-				TruthNamespace:  helper.TruthNamespace(),
-				ServiceAccounts: []string{render.PolicyRecommendationName},
-				KeyPairOptions: []rcertificatemanagement.KeyPairOption{
-					rcertificatemanagement.NewKeyPairOption(policyRecommendationKeyPair, true, true),
-				},
-				TrustedBundle: trustedBundle,
-			}),
-		)
+		if !r.multiTenant {
+			trustedBundleRW := certificateManager.CreateTrustedBundle(managerInternalTLSSecret, linseedCertificate)
+			components = append(components,
+				rcertificatemanagement.CertificateManagement(&rcertificatemanagement.Config{
```

Comment:
Could the `certificatemanagement.Config` be factored out of the if-else since everything is identical except `TrustedBundle`? That way it's more clear what exactly is being set conditionally and avoids issues with the 2 cases diverging when they shouldn't.